app/javascript/react/screens/App/Settings/SettingsActions.js

```diff
@@ -75,3 +80,36 @@ export const conversionHostWizardExitedAction = () => dispatch => {
     dispatch(reset(formName));
   });
 };
+
+export const setHostToDeleteAction = host => ({
+  type: SET_V2V_CONVERSION_HOST_TO_DELETE,
+  payload: host
+});
+
+export const showConversionHostDeleteModalAction = () => ({
+  type: SHOW_V2V_CONVERSION_HOST_DELETE_MODAL,
+  payload: true
+});
+
+export const hideConversionHostDeleteModalAction = () => ({
+  type: HIDE_V2V_CONVERSION_HOST_DELETE_MODAL,
+  payload: false
+});
+
+export const yesToDeleteConversionHostAction = () => ({
+  type: YES_TO_DELETE_CONVERSION_HOST_AND_HIDE_MODAL
+});
+
+export const deleteConversionHostAction = host => dispatch =>
+  dispatch({
+    type: DELETE_V2V_CONVERSION_HOST,
+    payload: new Promise((resolve, reject) => {
+      API.post(`/api/conversion_hosts/${host.id}`, {
```

Comment:
One more thing I realized here while working on my own PR: we should also pass the base of this URL in as an argument to the action creator along with `host`, and `API.post(\`${url}/${host.id}\`)`. We've been following the pattern of declaring the URLs for API requests as a prop on the view that calls the action creator, and defining them in `defaultProps`... a weird pattern, I know, I would have preferred to put them in constants... but that's what we've got for now. We probably want a `deleteConversionHostUrl` prop on ConversionHostsSettings. For example: https://github.com/ManageIQ/manageiq-v2v/blob/master/app/javascript/react/screens/App/Settings/screens/ConversionHostsSettings/ConversionHostsSettings.js#L83

Reply:
We should probably also use the `const uri = new URI(...)` and `uri.toString()` pattern we have elsewhere, for consistency.

Reply:
OK, done.